modules/cnf-topology-aware-lifecycle-manager-about-cgu-crs.adoc

```diff
@@ -55,6 +55,7 @@ After {cgu-operator} completes a cluster update, the cluster does not update aga
 * The `clusters` field specifies a list of clusters to update.
 * The `canaries` field specifies the clusters for canary updates.
 * The `maxConcurrency` field specifies the number of clusters to update in a batch.
+* The `actions` field specifies the actions that {cgu-operator} takes when the `ClusterGroupUpgrade` CR is enabled and after it is completed.
```

Comment:
after policy remediation is completed on clusters? Not sure that's clear enough to show that the labelling actions are done on a pre cluster basis.